common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -744,7 +747,12 @@ def create_response_element(self, **kwargs):
                 hintgroup_element.set("hintfn", hint_fn)
 
         for additional_answer in additional_answers:
-            etree.SubElement(response_element, "additional_answer").text = additional_answer
+            additional_node = etree.SubElement(response_element, "additional_answer")  # pylint: disable=no-member
```

Comment:
what is the pylint error you're disabling on this line? Why does it exist? (seems scary)